misc/helm-charts/operator/values.yaml

```diff
@@ -31,6 +31,11 @@ operator:
     environmentdIAMRoleARN: ""
     # awsAccountID is required when cloudProvider is "aws"
     awsAccountID: ""
+    # To provide additional disk to clusterd pods, configure a CSI plugin in
+    # your kubernetes cluster, and put its storage class here. Note that
+    # networked storage (EBS volumes, for instance) is unlikely to work well
+    # here.
```

Comment:
Perhaps we should explain that this should be high IOPS storage, and maybe recommend https://github.com/openebs/lvm-localpv on local NVME drives as a known working configuration?